packages/avs-bot/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "avs-bot",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
+  "dependencies": {
+    "@wireapp/avs": "6.0.28",
+    "@wireapp/core": "16.9.12",
+    "axios": "0.19.2",
```

Comment:
Dependency should be already transitively present because of the core

Reply:
https://github.com/wireapp/wire-web-packages/pull/3070/commits/52e36bfbe9125d9c35c570a20ab27a5512777438